src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentForDay = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  if (days >= 7) {
+    return days * rentForDay - bigDiscount;
+  }
+
+  if (days >= 3) {
+    return days * rentForDay - smallDiscount;
+  }
+
+  return days * 40;
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Save `days * rentForDay` in some const

```suggestion
    return days * rentForDay - bigDiscount; // days * rentForDay = const
  }

  if (days >= 3) {
    return days * rentForDay - smallDiscount; // days * rentForDay = const
  }

  return days * rentForDay; // const
```